integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInPV.java

```diff
@@ -296,6 +299,33 @@ void testIstioDomainHomeInPv() {
     } else {
       logger.info("Skipping WebLogic console in WebLogic slim image");
     }
+    
+    if (isWebLogicPsuPatchApplied()) {
+      String curlCmd2 = "curl -j -sk --show-error --noproxy '*' "
+          + " -H 'Host: " + domainNamespace + ".org'"
+          + " --user " + ADMIN_USERNAME_DEFAULT + ":" + ADMIN_PASSWORD_DEFAULT
+          + " --url http://" + K8S_NODEPORT_HOST + ":" + istioIngressPort
+          + "/management/weblogic/latest/domainRuntime/domainSecurityRuntime?" 
+          + "link=none";
+
+      ExecResult result = null;
```

Comment:
Same comment as above on line 264. Consider pulling this code into `executeCommand` method below since it's repeated multiple times.

```
private ExecResult executeCommand(String cmd) {
  logger.info("curl command {0}", cmd);
  return assertDoesNotThrow( () -> exec(cmd, true));
}
```